content/08.methods.md

```diff
@@ -45,11 +55,11 @@ Drafted sub-sections, edited the manuscript, reviewed pull requests, and coordin
 
 ### Competing interests
 
-A.K. is on the Advisory Board of Deep Genomics Inc.
-E.F. is a full-time employee of GlaxoSmithKline.
-The remaining authors have no competing interests to declare.
-
-The authors last reviewed and approved this competing interests statement on May 26, 2017.
+|Author|Competing Interests|Last Reviewed|
+|---|---|---|
+{% for author in manubot.authors %}
+|{{author.name}}|{{author.coi.string}}|{{author.coi.last-approved}}|
```

Comment:
We'll have to change `last-approved` to `last_approved` here an in the metadata file. These are treated as Python variable names so they can't have `-` characters.

Do you prefer author names or initials in the table?  We have access to both.